tests/test_collection.py

```diff
@@ -9,6 +9,28 @@
 from tests.helpers import assert_raises, extract, generate_col_name
 
 
+def test_get_collection_information(client):
+    # Test get information about collection
+    collection_name = generate_col_name()
+    client.create_collection(collection_name)
```

Comment:
I think we can reuse "col" collection here but before that just make sure to useFabric "tst_fabric_name"